autopts/ptsprojects/zephyr/csip.py

```diff
@@ -70,7 +70,7 @@ def set_pixits(ptses):
     pts2.set_pixit("CSIP", "TSPX_delete_link_key", "FALSE")
     pts2.set_pixit("CSIP", "TSPX_pin_code", "0000")
     pts2.set_pixit("CSIP", "TSPX_use_dynamic_pin", "FALSE")
-    pts2.set_pixit("CSIP", "TSPX_delete_ltk", "FALSE")
+    pts2.set_pixit("CSIP", "TSPX_delete_ltk", "TRUE")
```

Comment:
Maybe changes in this file should be in separate commit

Reply:
Done